lib/nrf_modem_lib/trace_backends/rtt/Kconfig

```diff
@@ -19,4 +19,10 @@ config NRF_MODEM_LIB_TRACE_BACKEND_RTT_BUF_SIZE
 	int "RTT buffer size"
 	default 1024
 
+config NRF_MODEM_LIB_TRACE_BACKEND_RTT_SKIP_IF_FIFO_FULL
```

Comment:
can this be default Y?

Reply:
When this Kconfig option is set the application can drop modem trace data. I think it should be disabled by default, and enabled only for those that need it for power consumption reasons.